pkg/apis/autoscaling/v1alpha1/metricsproducer.go

```diff
@@ -33,7 +33,7 @@ type MetricsProducerSpec struct {
 	ReservedCapacity *ReservedCapacitySpec `json:"reservedCapacity,omitempty"`
 	// ScheduledCapacity produces a metric according to a specified schedule.
 	// +optional
-	ScheduledCapacity *ScheduledCapacitySpec `json:"scheduledCapacity,omitempty"`
+	ScheduledCapacity *Schedule `json:"scheduledCapacity,omitempty"`
```

Comment:
For me, just `schedule` for the json does not scream MetricsProducer. I like scheduledCapacity here more.

Reply:
Agree with Ellis, scheduleCapacity in Json doesn't sound correct. Since, this is user facing it might be confusing to some.

Reply:
scheduledCapacity in any sense doesn't make sense, specifically because it has nothing to do with capacity. It could be used to schedule scaling for pods, nodes, birds, shoes, etc. Unless you have good reason to diverge, I'd keep the user facing field names and internal field names more or less the same. If "scheduled metrics producer" sounds odd to you, what sounds more correct?